kategory-docs/docs/docs/typeclasses/applicative/README.md

```diff
@@ -109,34 +109,34 @@ Option.applicative().map2Eval(Option(1), Eval.later { Option("x") }, { z: Tuple2
 
 ### Laws
 
-Kategory provides [`ApplicativeLaws`](/docs/typeclasses/laws#applicativelaws) in the form of test cases for internal verification of lawful instances and third party apps creating their own Applicative instances.
+Kategory provides [`ApplicativeLaws`]({{ '/docs/typeclasses/laws#applicativelaws' | relative_url }}) in the form of test cases for internal verification of lawful instances and third party apps creating their own Applicative instances.
 
 #### Creating your own `Applicative` instances
 
-[Kategory already provides Applicative instances for most common datatypes](#datatypes) both in Kategory and the Kotlin stdlib. 
+[Kategory already provides Applicative instances for most common datatypes](#datatypes) both in Kategory and the Kotlin stdlib.
```

Comment:
this URL actually does not work, where should point?

Reply:
We can get rid of the link